src/components/InfommatoionEmployee/InformatoionEmployee.jsx

```diff
@@ -0,0 +1,67 @@
+import './InformatoionEmployee.scss';
+
+import { useState } from 'react';
+
+import PersonalDataForm from '../PersonalDataForm/PersonalDataForm';
+import CurrentDataForm from '../CurrentDataForm/CurrentDataForm';
+
+export default function InformatoionEmployee() {
+  // const activeLi = 'current';
+
+  const [activeLi, setActiveli] = useState('personal');
+
+  function handlePersonalLi() {
+    setActiveli('personal');
+  }
+
+  function handleCurrentLi() {
+    setActiveli('current');
+  }
+
+  function handleArchiveLi() {
+    setActiveli('current');
```

Comment:
```suggestion
    setActiveli('archive');
```
Рекомендую делать все по фичам. Чтобы за раз делать какую-нибудь (самую важную на текущий момент) фичу полностью: и ее верстку, и интеграцию с бэком, и логику, и деплой. Таким образом можно будет полностью проверить работу каждой части в действии, а не только думать "раз в коде так написано, значит будет работать"

Нельзя доверять только тому, что в коде написано. Вернее, нельзя доверять собственному пониманию кода. Я думаю у вас уже были ситуации когда в коде вроде все правильно, а не работает. Поэтому надо проверять каждый кусочек логики, который ты написал. Мое наблюдение такое, что чем опытнее девелопер, тем меньше он доверяет тому, что написано (в первую очередь им самим), и тем тщательнее проверяет.